torch/testing/_deprecated.py

```diff
@@ -84,3 +86,16 @@ def assert_allclose(
         check_is_coalesced=False,
         msg=msg or None,
     )
+
+
+for name in _dtype_getters.__all__:
+    if name.startswith("_"):
+        continue
+
+    fn = getattr(_dtype_getters, name)
+    globals()[name] = warn_deprecated(f"The unparametrized call can be replaced with {fn()}")(fn)
```

Comment:
Should we include a reason as to why we deprecate / remove the dtype getters? Since there is no RFC for it, it could seem arbitrary.